{{cookiecutter.project_slug}}/Makefile

```diff
@@ -11,20 +11,17 @@ webbrowser.open("file://" + pathname2url(os.path.abspath(sys.argv[1])))
 endef
 export BROWSER_PYSCRIPT
 
-define PRINT_HELP_PYSCRIPT
-import re, sys
-
-for line in sys.stdin:
-	match = re.match(r'^([a-zA-Z_-]+):.*?## (.*)$$', line)
-	if match:
-		target, help = match.groups()
-		print("%-20s %s" % (target, help))
-endef
-export PRINT_HELP_PYSCRIPT
 BROWSER := python -c "$$BROWSER_PYSCRIPT"
 
 help:
-	@python -c "$$PRINT_HELP_PYSCRIPT" < $(MAKEFILE_LIST)
+	echo "import re, sys" > .print_help_script.py
+	echo "for line in sys.stdin:" >> .print_help_script.py
+	echo "    match = re.match(r'^([a-zA-Z_-]+):.*?## (.*)$$', line)" >> .print_help_script.py
+	echo "    if match:" >> .print_help_script.py
+	echo "        target, help = match.groups()" >> .print_help_script.py
+	echo "        print('%-20s %s' % (target, help))" >> .print_help_script.py
+	@python .print_help_script.py < $(MAKEFILE_LIST)
```

Comment:
I really hope not, there has to be a better way, I just couldn't work out how. You are right, looks like only an issue in Appveyor build. I'll take another look.